### DISCUSSION TITLE
Wildcard domains with SSG

### DISCUSSION THREAD

**Discussion by aaronhayes** [vercel/next.js#20051] [Help]
Hi all 👋 ,

I'm trying to use [wildcard domains](https://vercel.com/blog/wildcard-domains) with static site generation. The content on each page doesn't change frequently - perfect for static generation. How can I know which subdomain I'm working with inside `getStaticProps` ? 

For example, I need to pass different props to these two pages.

1. `client.mydomain.com/about`
2. `client2.mydomain.com/about`

**Comment by tiivik** [comment]
Have exactly the same issue when wanting to implement caching based on subdomains.

**Comment by SleepWalker** [comment]
A similar discussion with updates regarding next@10.2 and new redirect features:

https://github.com/vercel/next.js/discussions/12848#discussioncomment-680225
